### PR TITLE
TR-894: Add rtl for basic lists

### PR DESCRIPTION
**Pull request by meefox**
- [ ] HOLD MERGE 

Related to https://oat-sa.atlassian.net/browse/TR-894
2/3

Depends
https://github.com/oat-sa/tao-core/pull/3096

### CODE REVIEW

**Review by marpesia — CHANGES_REQUESTED** [review]
It is only working when the layout it on RTL mode, but not when only the item is on RTL mode

layout_ltr
layout_rtl

File: scss/inc/base/_list-style.scss [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ul, ol {
     padding-left: 40px;
+    padding-right: 3px;
```
Comment (marpesia) [inline scss/inc/base/_list-style.scss:3]: I had here the same issue as you on my ticket.
I solved it with 
`body[dir="rtl"]`

Comment (meefox) [reply scss/inc/base/_list-style.scss:3]: changes above should fix the issue

**Review by marpesia — APPROVED** [review]
Good job!

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ]  Commits messages are meaningful
- [ ]  Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: scss/inc/_base.scss [hunk 2]
```diff
@@ -478,19 +478,17 @@ body {
         font-family: $regularFont;
         font-size: 1.4rem !important;
     }
+}
 
+// File may be used standalong, this is why you can not move imports to basic.scss easaly.
```
Comment (jsconan) [inline scss/inc/_base.scss:483]: **chore:** Small typos...
```suggestion
// File may be used standalone, this is why you cannot move imports to basic.scss easily.
```

**Review by jsconan — APPROVED** [review]
The code looks ok. I didn't see any impact on other places.

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Comment by meefox** [comment]
Simplified by https://github.com/oat-sa/extension-tao-itemqti/pull/1871

**Pull request closed without merging**